Makefile

```diff
@@ -1,3 +1,6 @@
+# Y - in response to Would you like to add the OpenAPI addon?
+openapi_addon_prompt = Y\n
+
 # Y - in response to Would you like to add the Github addon?
```

Comment:
I think it's better if we stop using there variables (common_addonx_prompts, web/api, ...) and instead just use a string and have the comments before:

```
# Y - in response to Would you like to add the Github addon?
# Y - in response to Would you like to add the Open[API](https://docs.github.com/) addon?
# Y - in response to Would you like to add the SemaphoreCI addon?
# Y - in response to Would you like to add the Nginx addon?
# Y - in response to Would you like to add the Phrase addon?
# Y - in response to Would you like to add the Devise addon?
# Y - in response to Would you like to add the Bootstrap addon?
# Y - in response to Would you like to add the Slim Template Engine addon?
# Y - in response to Would you like to add the Hotwire addon?
create_web:
	printf "Y\nY\nY\nY\nY\nY\nY\nY\nY\n" | rails new $(APP_NAME) -m ./template.rb -T ${OPTIONS}
```

(and duplicate that for the API)

Because how the API includes the OpenAPI addons, using core makes it hard to read. Also let's ensure to keep the new ordering (GitHub first)

Thanks :)

(that should be my last comment :bow:)

Reply:
Just resolved your last comment in 5695671 ✅ 😛